boards/st/stm32h7s78_dk/stm32h7s78_dk_stm32h7s7xx_ext_flash_app.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: stm32h7s78_dk/stm32h7s7xx/ext_flash_app
+name: ST STM32H7S78 Discovery Kit with App in Ext Flash
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+ram: 640
```

Comment:
`ram: 128` as per `sram0` current size?
(Maybe a appended commit to fix *boards/st/stm32h7s78_dk/stm32h7s78_dk.yaml* the same way)

Reply:
yes, good catch.